lib/tapioca/compilers/dsl/active_record_columns.rb

```diff
@@ -359,7 +359,10 @@ def handle_unknown_type(column_type)
 
         def lookup_return_type_of_method(column_type, method)
           signature = T::Private::Methods.signature_for_method(column_type.instance_method(method))
-          signature.return_type.to_s if signature && !signature.return_type.to_s.include?("T::Private::Types")
+          return unless signature
+
+          signature.return_type.to_s if signature.return_type.to_s != "<VOID>" &&
+              signature.return_type.to_s != "<NOT-TYPED>"
```

Comment:
This is fine but we keep doing the `.to_s` over and over again, which is not cool. I will suggest:
```suggestion
          return_type = signature.return_type.to_s
          return_type if return_type != "<VOID>" && return_type != "<NOT-TYPED>"
```

Reply:
Basically, the `.to_s` call could potentially be very expensive so best to not repeat it.